framework/src/media/audio/audio_manager.c

```diff
@@ -1614,8 +1614,21 @@ audio_manager_result_t find_stream_in_device_with_process_type(device_process_ty
 	audio_card_info_t *card;
 
 	if (type == AUDIO_DEVICE_PROCESS_TYPE_NONE) {
-		meddbg("Type is AUDIO_DEVICE_PROCESS_TYPE_NONE\n");
-		return AUDIO_MANAGER_INVALID_PARAM;
+		medvdbg("Type is AUDIO_DEVICE_PROCESS_TYPE_NONE\n");
+		for (j = 0; j < CONFIG_AUDIO_MAX_DEVICE_NUM; j++) {
```

Comment:
This is temp code, please put /* TODO will be implemented later based on audio subsystem's implemantation */ or some other comments.

Reply:
Added comments